01-numpy.md

```diff
@@ -490,8 +490,16 @@ Visualization deserves an entire lecture (or course) of its own,
 but we can explore a few features of Python's `matplotlib` library here.
 While there is no "official" plotting library,
 this package is the de facto standard.
+
 First,
-we will import the `pyplot` module from `matplotlib`
+we need to to run a special IPython notebook command
+to ensure that our figures display within the notebook itself.
+
+~~~ {.python}
+% matplotlib inline
```

Comment:
While we mention running this in an IPython notebook at the outset, these lessons are designed to run in a normal Python interpreter. Perhaps we can put this in a callout? It might be a good thing to have specific styling for IPython specific callouts... I made issue #119 for that :)